Formula/feroxbuster.rb

```diff
@@ -0,0 +1,24 @@
+class Feroxbuster < Formula
+  desc "Fast, simple, recursive content discovery tool written in Rust"
+  homepage "https://epi052.github.io/feroxbuster"
+  url "https://github.com/epi052/feroxbuster/archive/v2.3.3.tar.gz"
+  sha256 "5a244d1614fb9dc476e9be8bd1cdbb33e0b07a1fcb7285e911ae6c77a44c7327"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  def caveats
+    <<~EOS
+      To use feroxbuster, additional wordlists are required.
+      A good source is: https://github.com/danielmiessler/SecLists
+    EOS
+  end
+
+  test do
+    system bin/"feroxbuster", "-h"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Yes, you're absolutely right. This is not a great test. The problem is the the tool runs against a website so I thought it would create a dependency on some random website to make the test pass.

Maybe it could run it against its own docs page so it would at least rely on its own sites. It also needs a word list to function but it's probably best to generate one inside the test to keep the test short and simple.

Another idea would be to spin up a local dev server as part of the test. Using python would make this very easy I think. Then it wouldn't rely on any external site at all.
You don't happen to know any other formula that does something similar so that I could copy a few tricks?

Reply:
Spinning up a local server seems fine as long as it's not too complicated. Looking for formulae that do `fork do` or `spawn` might be a good start here, there are quite a few examples.